plugins/modules/elb_network_lb.py

```diff
@@ -283,6 +294,13 @@
                             returned: when state is present
                             type: str
                             sample: ""
+                alpn_policy:
+                    description: The name of the Application-Layer Protocol Negotiation (ALPN) policy.
+                    returned: when state is present
+                    type: list
+                    elements: str
+                    version_added: 7.1.0
+                    sample: ["HTTP1Only", "HTTP2Only"]
```

Comment:
Wouldn't the response only have one item in the list?

Reply:
yes, updated